File: projects/android/Sample/app/src/main/java/com/ezored/sample/events/EventNetworkStateChanged.java
package com.ezored.sample.events;

public class EventNetworkStateChanged extends BaseEvent {

    private int type;

    private boolean connected = true;

    public EventNetworkStateChanged(boolean connected, int type) {
        this.type = type;
        this.connected = connected;
    }

    public int getType() {
        return type;
    }

    public boolean isConnected() {
        return connected;
    }

}

File: projects/android/Sample/app/src/main/java/com/ezored/sample/events/EventNetworkStateChanged.kt
package com.ezored.sample.events

class EventNetworkStateChanged : BaseEvent {

    var isConnected = true
    var type: Int = 0

    constructor(isConnected: Boolean, type: Int) : super() {
        this.isConnected = isConnected
        this.type = type
    }

}
